### ISSUE TITLE
import for command with backticks

### ISSUE THREAD

**Issue by mjhsieh** [jcsalterego/historian#12]
Have you seen this before?
```
[user@centos7 ~]$ cat .bash_history
echo `whoami`
[user@centos7 ~]$ local/libexec/hist import
Error: /home/user/.bash_history line 1: expected 1 columns of data but found 3
Imported 0 item(s).
```

**Comment by jcsalterego** [comment]
@mjhsieh Yes! That was a problem in the first version and should have been resolved in #1 and version 0.0.2. Can you try getting the latest version and see if it's still a problem?

**Comment by mjhsieh** [comment]
Thank you Jerry! I can confirm that this problem has been solved in the latest version.
